ai/unet/Model Training.py

```
@@ -0,0 +1,528 @@
+        with torch.amp.autocast('cuda', enabled=use_amp):
+            out  = model(images)
+            loss = criterion(out, masks)
+
+        scaler.scale(loss).backward()
+        scaler.unscale_(optimizer)
+        torch.nn.utils.clip_grad_norm_(model.parameters(), max_norm=1.0)
+        scaler.step(optimizer)
+        scaler.update()
+
+        d, c, _ = dice_scores(out, masks)
+        tot_loss += loss.item(); tot_disc += d; tot_cup += c
```

Ruff (0.15.7)

[error] Multiple statements on one line (semicolon)

(E702)